generators/entity/index.js

```diff
@@ -35,9 +34,16 @@ var fieldNamesUnderscored = ['id'];
 var fieldNameChoices = [], relNameChoices = []; // this variable will hold field and relationship names for question options during update
 var databaseType;
 var prodDatabaseType;
-const INTERPOLATE_REGEX = constants.INTERPOLATE_REGEX;
-const RESOURCE_DIR = constants.RESOURCE_DIR;
-const ANGULAR_DIR =  constants.ANGULAR_DIR;
+
+const constants = require('../generator-constants'),
+    INTERPOLATE_REGEX = constants.INTERPOLATE_REGEX,
+    CLIENT_MAIN_SRC_DIR = constants.CLIENT_MAIN_SRC_DIR,
+    CLIENT_TEST_SRC_DIR = constants.CLIENT_TEST_SRC_DIR,
+    ANGULAR_DIR =  CLIENT_MAIN_SRC_DIR + 'app/',
```

Comment:
I thought about putting `app` and `content` in constants, but it would have made this PR a lot more complicated (client templates, mostly). I would propose doing this separately, if you think it adds value... I was mainly trying to facilitate the layout change discussed earlier, for now.